arquillian-ape-sql/standalone/dbunit/src/main/java/org/arquillian/ape/rdbms/dbunit/DbUnitOptions.java

```diff
@@ -0,0 +1,100 @@
+package org.arquillian.ape.rdbms.dbunit;
+
+import java.util.Collection;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Set;
+
+public class DbUnitOptions implements Map<String, Object> {
```

Comment:
We do have DBUnit configuration for `arquillian.xml` - isn't it something we can use?